keepalived/vrrp/vrrp_print.c

```diff
@@ -418,3 +421,182 @@ vrrp_print_stats(void)
+			json_object_new_int(vrrp->dont_track_primary));
+		json_object_object_add(json_data, "skip_check_adv_addr",
+			json_object_new_int(vrrp->skip_check_adv_addr));
+		json_object_object_add(json_data, "strict_mode",
+			json_object_new_int(vrrp->strict_mode));
+		json_object_object_add(json_data, "vmac_flags",
+			json_object_new_int(vrrp->vmac_flags));
+		json_object_object_add(json_data, "vmac_ifname",
+			json_object_new_string(vrrp->vmac_ifname));
+		json_object_object_add(json_data, "master_priority",
+			json_object_new_int(vrrp->master_priority));
+		json_object_object_add(json_data, "last_transition",
+			json_object_new_int(vrrp->last_transition.tv_sec));
+		json_object_object_add(json_data, "garp_delay",
+			json_object_new_int(vrrp->garp_delay));
+		json_object_object_add(json_data, "garp_refresh",
+			json_object_new_int(vrrp->garp_refresh.tv_sec));
+		json_object_object_add(json_data, "garp_refresh_timer",
+			json_object_new_int(vrrp->garp_refresh_timer.tv_sec));
+		json_object_object_add(json_data, "garp_rep",
+			json_object_new_int(vrrp->garp_rep));
+		json_object_object_add(json_data, "garp_refresh_rep",
+			json_object_new_int(vrrp->garp_refresh_rep));
+		json_object_object_add(json_data, "garp_lower_prio_delay",
+			json_object_new_int(vrrp->garp_lower_prio_delay));
+		json_object_object_add(json_data, "garp_pending",
+			json_object_new_boolean(vrrp->garp_pending));
+		json_object_object_add(json_data, "gna_pending",
+			json_object_new_boolean(vrrp->gna_pending));
+		json_object_object_add(json_data, "garp_lower_prio_rep",
+			json_object_new_int(vrrp->garp_lower_prio_rep));
+		json_object_object_add(json_data, "lower_prio_no_advert",
+			json_object_new_int(vrrp->lower_prio_no_advert));
+		json_object_object_add(json_data, "higher_prio_send_advert",
+			json_object_new_int(vrrp->higher_prio_send_advert));
+		json_object_object_add(json_data, "vrid",
+			json_object_new_int(vrrp->vrid));
+		json_object_object_add(json_data, "base_priority",
+			json_object_new_int(vrrp->base_priority));
+		json_object_object_add(json_data, "effective_priority",
+			json_object_new_int(vrrp->effective_priority));
+		json_object_object_add(json_data, "vipset",
+			json_object_new_boolean(vrrp->vipset));
+
+		//Virtual IPs are stored in a list
+		for (f = LIST_HEAD(v); f; ELEMENT_NEXT(f)) {
+			ip_address_t *vip = ELEMENT_DATA(f);
+			char ipaddr[INET6_ADDRSTRLEN];
+			inet_ntop(vrrp->family, &(vip->u.sin.sin_addr.s_addr),
+				ipaddr, INET6_ADDRSTRLEN);
+			json_object_array_add(vips,
+				json_object_new_string(ipaddr));
+		}
+		json_object_object_add(json_data, "vips", vips);
+
+		json_object_object_add(json_data, "promote_secondaries",
+			json_object_new_boolean(vrrp->promote_secondaries));
+		json_object_object_add(json_data, "evip_add_ipv6",
+			json_object_new_boolean(vrrp->evip_add_ipv6));
+		json_object_object_add(json_data, "master_adver_int",
+			json_object_new_int(vrrp->master_adver_int));
```

Comment:
This should be `json_object_new_double(vrrp->master_adver_int / TIMER_HZ_FLOAT))` so that it is reported in seconds.